# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/res/debug_font.glsl
# Commit: 0edb6b6f17b2
# Full Hash: 0edb6b6f17b24c737d053df19730e9c9e61e17ca
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-18 04:07:40
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r ee0de0eac18d -r 0edb6b6f17b2 gfx/wr/webrender/res/debug_font.glsl
--- a/gfx/wr/webrender/res/debug_font.glsl	Thu Dec 17 22:26:08 2020 +0000
+++ b/gfx/wr/webrender/res/debug_font.glsl	Thu Dec 17 22:26:10 2020 +0000
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#define WR_FEATURE_TEXTURE_2D
+
 #include shared,shared_other
 
 varying vec2 vColorTexCoord;
@@ -22,7 +24,7 @@
 
 #ifdef WR_FRAGMENT_SHADER
 void main(void) {
-    float alpha = texture(sColor0, vec3(vColorTexCoord.xy, 0.0)).r;
+    float alpha = texture(sColor0, vColorTexCoord).r;
     oFragColor = vColor * alpha;
 }
 #endif